src/site/get-started/change-the-image-assets-path.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: layouts/markdown-content-page.njk
+title: Change the image assets path
+description: This guide will help you build the CSS to point to your images folder.
+tags: get-started
+order: 300
```

Comment:
Seems to be last in the prototype/on Confluene (i.e. `500`) unless you've heard differently.